src/qililab/yaml.py

```diff
@@ -30,9 +31,64 @@ def deque_constructor(constructor, node):
     return deque(constructor.construct_sequence(node))
 
 
+# Lambda representer: convert lambda to a string
+def lambda_representer(representer, data):
```

Comment:
should we add typings?

Reply:
Not needed. These functions are not called by humans, but by YAML library. And all their parameters' types are `Any` which is the implicit value if you don't provide a specific type.